tensorflow/python/ops/init_ops_v2.py

```diff
@@ -636,7 +636,7 @@ def lecun_normal(seed=None):
       seed: A Python integer. Used to seed the random generator.
```

Comment:
Can you change these as well to make it consistent indentation within the docstring please

Reply:
thanks for the feedback @tamaranorman , i have changed the indentation on seed lines indicated and the ones below.